### DISCUSSION TITLE
自创群

### DISCUSSION THREAD

**Discussion by komhshan** [share121/inter-knot#5410] [委托]
![qrcode_1727275646591](https://github.com/user-attachments/assets/02f607d5-8944-4f60-b300-298972900fdc)
群里缺人，来点人呗

**Comment by Zou-pig** [comment]
这是原神群吗

**Reply by komhshan** [reply]
是的

**Reply by Zou-pig** [reply]
好滴<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/829f0a1d533765b80769dd1a38dc338c_9182108590963360703.png" alt="阿姬惊讶" width="75" >
